shuup_setup_utils/resource_building.py

```diff
@@ -70,7 +71,8 @@ def _find_package_json_dirs(self):
         items = excludes.walk_excl(self.root_directory)
         for (dirpath, dirnames, filenames) in items:
             if 'package.json' in filenames:
-                if 'generated_resources.txt' in filenames:
+                package = json.load(open(os.path.join(dirpath, "package.json")))
```

Comment:
still check for `'generated_resources.txt' in filenames` for backwards compatibility